index.css

```diff
@@ -385,6 +348,150 @@ li {
   margin-top: 1em;
 }
 
+.modal {
+  display: none;
+  position: fixed;
+  border-radius: 20px;
+  z-index: 10;
+  top: 0;
+  width: 90%;
+  overflow: auto;
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(0, 0, 0, 0.4);
+  left: 0;
+  margin: 0;
+}
+
+.modal-buttons {
+  display: flex;
+  justify-content: center;
+}
+
+.close-modal {
+  display: inline !important;
+  color: var(--title-color);
+  font-size: x-large;
+  margin: 0;
+}
+
+.modal-title-close h2 {
+  width: fit-content;
+  margin: 0;
+  display: inline-block;
+}
+
+.modal-title-close {
+  display: flex;
+  align-items: baseline;
+  justify-content: space-between;
+  width: 100%;
+  padding: 0;
+  margin: 0;
+  height: 50px;
+}
+
+.modal-top {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  gap: 1em;
+}
+
+@media (min-width: 375px) {
+  .modal {
+    display: none;
+    position: fixed;
+    border-radius: 20px;
+    z-index: 10;
+    top: 0;
+    width: 100vw;
+    height: 100vh;
+    overflow: auto;
+    background-color: rgb(0, 0, 0);
+    background-color: rgba(0, 0, 0, 0.4);
+    left: 0;
+    margin: 0;
+    padding: 0;
+    animation: popupSlideDown 0.3s;
+  }
+
+  @keyframes popupSlideDown {
+    from {
+      transform: translateY(-50%);
+    }
+
+    to {
+      transform: translateY(0);
+    }
+  }
+
+  .modal-desk {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    height: fit-content;
+    width: fit-content;
+    margin: auto;
+    background-color: white;
+    border-radius: 25px;
+    padding: 25px;
+    z-index: 20;
+  }
```

Comment:
- [ ] Although you did an excellent job designing the popup window :+1:, it would be better if you could ensure that the popup window fits the size of the screen in the desktop version, without the need for a scrollbar. This means that the content within the popup window should be adjusted to the dimensions of the screen so that users can view all of the information without having to scroll up or down or side to side. This will improve the user experience and make it easier for them to navigate through the popup window.

**Your design**

![Screenshot from 2023-03-09 06-57-36](https://user-images.githubusercontent.com/105117832/224017926-e61d420c-486c-459e-bac9-eec029ff83b2.png)

**Figma**

![Screenshot from 2023-03-09 07-01-02](https://user-images.githubusercontent.com/105117832/224017997-fbb9b834-10e2-45f1-8c37-442a63f27ff3.png)